### PR TITLE
Update wording on organism components

### PR DESCRIPTION
**Pull request by michaelcarter**

### CODE REVIEW

File: working-with-react.md [hunk 1]
```diff
@@ -35,8 +35,8 @@ When modifying existing user interface code, existing user interface elements th
...
- - **Organism React Components:** For now, keep these in the repository of the project you're working on. Very shortly we'll be setting up a repository for all of our "organism" react components, at which point we can move components and their associated tests across into a shared area.
-
+ - **Organism React Components:** These should be built as standalone modules under their own repository. In order to assist with this we've created a [React Component Template](https://github.com/holidayextras/react-component-template) project which you can clone and push up under a new repository with your component code. You should use this template unless there's a good reason not to, as it saves time writing boilerplate code and ensures all of our compoenent codebases are familiar.
```
Comment (lukehansell) [inline working-with-react.md]: compoenent > component

File: working-with-components.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+## Why should we use components?
+Building a library of reuseable components has several benefits:
+
+ - Once built, the shared components can be reused in a fraction of the time they took to build.
+ - Shared components provide a consistent look and feel to all of Holiday Extras' user interfaces.
+ - Shared components allow user interface improvements to be rapidly rolled out across our websites and all of our customers and partners to benefit in a fraction of the time they otherwise would.
+ - Simple reuseable components can be rapidly composed together to build more complex user interfaces.
+ - Reuseable components vastly reduce code duplication in the front end.
+
+In the short term there's an obvious implemntation cost to setting all this up, but taking a long term view and keeping the above in mind, the question is really *"Why wouldn't we use components?"*.
```
Comment (lukehansell) [inline working-with-components.md]: implemntation  > implementation

File: working-with-components.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+In short, we'll use them everywhere. Our goal is to be integrating reuseable components across all of our user interfaces. From our landing pages and search engines, to our web and mobile apps. In order to do this we'd like everyone to look at componentising user interfaces as they work on them.
+
+As for when to use components, the answer to this question can be broken down into two sections:
+
+### Writing new user interface code
+This should be componentised from the beginning, and this is the expectation for new pull requests. The UX/UI Team are on hand to offer advice and assistance with this if necessary.
+
+### Modifying existing user interface code
+When modifying existing user interface code, existing user interface elements that can be replaced by shared components we already have should be replaced. If there are no existing user interface components we should look at building some when time allows.
+
```
Comment (lukehansell) [inline working-with-components.md]: you could put a note in here about refactoring from the smallest components upwards. Always replace the end nodes of the tree (atoms) before attempting to refactor out molecules or organisms. Lesson learned from parking tile and wizard. Can like to Ryan Florence for more in depth info.

File: working-with-components.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+ - **Atom and Molecule React Components:** These go in the public [UI-Toolkit](https://github.com/holidayextras/ui-toolkit) project where they can be reused everywhere. This is because at the granular atomic and molecular level, components have applicability across all of the Hx platforms _as well as_ to the wider world. An easy comparison to make is to the Twitter Bootstrap project, it was originally built by Twitter, but was designed for everyone to use. In the same way, the UI-Toolkit project and everything that goes into it is open source.
+ - **Organism React Components:** These should be built as standalone modules under their own repository. In order to assist with this we've created a [React Component Template](https://github.com/holidayextras/react-component-template) project which you can clone and push up under a new repository with your component code. You should use this template unless there's a good reason not to, as it saves time writing boilerplate code and ensures all of our component codebases are familiar.
+ 
+If you're building a new React component and you're unsure about where it should go, talk to one of the SAs or the UXUI team.
```
Comment (lukehansell) [inline working-with-components.md]: also React gurus

**Comment by jackdclark** [comment]
:+1:

**Comment by Joezo** [comment]
Makes sense :)

**Comment by hx-markterry** [comment]
ok with me :star:

**Pull request merged**